addons/website_forum/views/forum_forum_templates_post.xml

```diff
@@ -594,5 +595,28 @@
     </div>
 </template>
 
+<template id="forum_related_posts" name="Related Posts">
+    <table class="o_wforum_table table mt-5" height="1">
+        <thead>
+            <tr class="d-none d-lg-table-row">
+                <th class="o_wforum_table_title">Related Posts</th>
+                <th class="o_wforum_table_posters"></th>
+                <th class="o_wforum_table_replies small fw-normal text-center text-muted">
+                    Replies
+                </th>
+                <th class="o_wforum_table_views small fw-normal text-center text-muted">
+                    Views
+                </th>
+                <th class="o_wforum_table_activity small fw-normal text-center">
```

Comment:
Not entirely sure why "Replies, Views" are text-muted but not "Activity".

Shouldn't we let them all non-muted? Maybe I'm missing something.